modules/sdk-coin-btc/src/btc.ts

```diff
@@ -34,4 +34,8 @@ export class Btc extends AbstractUtxoCoin {
   supportsBlockTarget(): boolean {
     return true;
   }
+
+  supportsLightning(): boolean {
+    return true;
```

Comment:
this is ok for now but maybe we can add a feature in `statics` for lightning support?